### PR TITLE
Terminology Change

### PR DESCRIPTION
**Pull request by rori4**
- related to Terminology/nomenclature in the platform (AdExNetwork/adex-platform#268)

a lot of things have been changed so I made it as a pull request. Also I have changed On identity to In Account as the total available balance is the one on channels and the one on the identity.

### CODE REVIEW

File: en-US.json [hunk 1]
```diff
@@ -700,50 +700,50 @@
...
 	"SELECT_PRIV_LEVEL": "Select privilegе level",
 	"SELECT_PRIV_LEVEL_HELPER_TXT": "Choose wisely",
 	"ACCOUNT_SET_IDENTITY_PRIVILEGE_SAVE_BTN": "Set privileges",
 	"ACCOUNT_ADVANCED_INFO_AND_ACTIONS": "Advanced",
 	"MANAGE_IDENTITY": "Manage idenity",
-	"ERR_IDENTITY_SET_ADDR_PRIV_NOTIFICATION": "Error setting identity privileges: ${0}",
-	"IDENTITY_SET_ADDR_PRIV_NOTIFICATION": "Identity privileges transactions sent!",
-	"INSUFFICIENT_BALANCE_FOR_FEES": "Insufficient balance for transaction fees. Identity avaialbe: ${0} ${1}, fees: ${2} ${3}",
+	"ERR_IDENTITY_SET_ADDR_PRIV_NOTIFICATION": "Error setting account privileges: ${0}",
+	"IDENTITY_SET_ADDR_PRIV_NOTIFICATION": "Account ID privileges transactions sent!",
```
Comment (simzzz) [inline en-US.json:730]: Why not just `Account`

File: en-US.json [hunk 2]
```diff
@@ -700,50 +700,50 @@
...
+	"INSUFFICIENT_BALANCE_FOR_FEES": "Insufficient balance for transaction fees. Account balance avaialbe: ${0} ${1}, fees: ${2} ${3}",
```
Comment (simzzz) [inline en-US.json:731]: There's a typo from before in `avaialbe`, should be `available`

File: en-US.json [hunk 3]
```diff
@@ -700,50 +700,50 @@
...
 	"PRIV_LEVEL_INFO_AND_FEES": "You will pay ${0} ${1} from idenitity for this transaction",
 	"PROP_SETADDR": "Address to set privilege",
 	"PROP_PRIVLEVEL": "Privilege level",
-	"ERR_IDENTITY_WITHDRAW_NOTIFICATION": "Error on withdrow from identity: ${0}",
+	"ERR_IDENTITY_WITHDRAW_NOTIFICATION": "Error on withdrow from account: ${0}",
```
Comment (simzzz) [inline en-US.json:735]: Another typo from before, `withdrow` should be `withdraw`

File: en-US.json (hunk 2 above)
Comment (rori4) [reply en-US.json:731]: Nice catch!

File: en-US.json (hunk 3 above)
Comment (rori4) [reply en-US.json:735]: Nice catch again

File: en-US.json (hunk 1 above)
Comment (rori4) [reply en-US.json:730]: Yes you are right

**Pull request merged**